actions.yaml

```diff
@@ -20,3 +20,7 @@ update-db:
   required:
     - current-version
     - target-version
+string-params:
+  description: |
+    Updates de database to change the instantiation and primitive params to a
+    string after the fix for bug1837
```

Comment:
Updates database to change NS params, VNF params and KDU params from dictionary to string. (https://osm.etsi.org/bugzilla/show_bug.cgi?id=1837)